.gitignore

```diff
@@ -1,8 +1,30 @@
+# Not sure what a .gitignore is?
+# See: https://git-scm.com/docs/gitignore
+
+# The first files are directly copied from Jekyll's first-party docs on `.gitignore` files:
+# https://jekyllrb.com/tutorials/using-jekyll-with-bundler/#commit-to-source-control
+
+# Ignore metadata generated by Jekyll
```

Comment:
```suggestion
# Ignore the default location of the built site, and caches and metadata generated by Jekyll
```
It might be worth noting that Jekyll allows configuration of a different location for the built site.